enarx-copy-labels-linked

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/python3
+# SPDX-License-Identifier: Apache-2.0
+
+from github import Github
+import common
+import itertools
+import os
+import re
+
+# We don't want to copy all labels on linked issues; only those in this subset.
+BLOCKED_LABELS = {
+    "meta",
+    "question",
+    "help wanted",
+    "good first issue",
+    "duplicate",
+}
+
+github = Github(os.environ['GITHUB_TOKEN'])
+
+# Get the Enarx org.
+org = github.get_organization('enarx')
+
+# Get the Planning board.
+planning = {p.name: p for p in org.get_projects(state='open')}['Planning']
+
+# Cache the Planning board to avoid redundant network requests.
+planning_cached = common.cache_board(planning)
+
+for issue in github.search_issues(f"org:enarx is:pr is:public is:open linked:issue"):
+    # Some attributes change when converting to PR; get them beforehand.
+    repo = issue.repository
+
+    # Convert to a PR.
+    pr = issue.as_pull_request()
+
+    # Get a list of related issues to the PR.
+    related = [repo.get_issue(i) for i in set(common.get_related_issues(pr))]
```

Comment:
```suggestion
    related = {repo.get_issue(i) for i in common.get_related_issues(pr)}
```

I'm not sure why you're converting the inner iterable to a set, but the outer comprehension is a list. Maybe I missed something.

Reply:
I converted to a `set()` because the output of `get_related_issues` could contain duplicates (e.g. if the PR submitter included `Resolves #10` in both the PR body and a commit message). Though I suppose that's something best fixed in that function.

Reply:
I've fixed the issue in `get_related_ issues`.